packages/editor/src/components/DataImport.jsx

```diff
@@ -26,6 +26,20 @@ import '../scss/data-import.scss'
 
 import '@cdc/core/styles/v2/components/data-designer.scss'
 
+const isSolrCsv = dataUrl => {
+  if (dataUrl.includes('wt=csv')) {
+    return true
+  }
+  return false
+}
+
+const isSolrJson = dataUrl => {
+  if (dataUrl?.includes('wt=json')) {
+    return true
+  }
+  return false
+}
+
```

Comment:
could you collapse this into one function: 

const getSolrExtension = (dataURL: string) => '.json' | '.csv' | null